### DISCUSSION TITLE
Jobs will not start

### DISCUSSION THREAD

**Discussion by chingc** [community/community#26543] [Actions]
<p>I’ve noticed this morning some of my jobs are stuck on “Starting workflow run”.  Most of them run, but a few just won’t start.  I’m using a mix of self-hosted runners and github-hosted runners and can see the behavior on both.</p>
<p>I also came across an error I’ve never seen before in a scheduled workflow run that ran overnight:<br>
“The request: 529700 was abandoned due to an infrastructure failure. Notification of assignment to a runner was never received.”</p>
<p>Is anyone else experiencing issues?  The GitHub Status page doesn’t show anything wrong right now, but I have a feeling it might show something soon.</p>

**Comment by dimisjim** [comment]
<p>experiencing the exact same issue here since at least 6 hours ago</p>

**Comment by chingc** [comment]
<p>I contacted support and they said there was an incident this morning that might have caused this problem and said they’ll get back to me.</p>

**Comment by dimisjim** [comment]
<p>Weird, then why isn’t it in the public status page?</p>
<p>anyway, thanks for sharing</p>

**Comment by chingc** [comment, marked as answer]
<p>They told me they had an incident affecting a very small number of jobs, so that’s probably why it never showed up on the github status page.  Anyway, github support was able to resolve my issues.  If you’re still suffering from this problem you should definitely <a href="https://support.github.com/contact" rel="noopener nofollow ugc">contact them directly</a>.  Support was pretty quick in getting back to me.</p>

**Comment by dimisjim** [comment]
<p>indeed it was resolved as well for us a few hours after. I also contacted them about it :+1:</p>
